### DISCUSSION TITLE
Unable to create an auto-renewing certificate for an Azure-hosted app

### DISCUSSION THREAD

**Discussion by kiwidyne** [rcl-ssl/docs#39] [General]
Whenever I try to create a certificate tied to an Azure application, I receive the following error: "Could not retrieve your Azure Subscription(s). Refer to the help link to ensure [Access Control] (https://docs.rclapp.com/authorization/access-control-user.html) is properly set up for your AAD account."

I've followed the directions. The permissions *appear* to be correct, but it never works. What I have been doing instead is to just keep creating standalone certificates every 90 days and manually installing them, which is a bit of a pain.

We can take this offline so I can send you more detailed account information in order to help you resolve this.

Thanks.

   -Rob

**Comment by rcladmin** [comment]
please email me with the details at support@rclapp.com
